com.microsoft.mrtk.graphicstools.unity/Runtime/Experimental/Magnifier/Scripts/MagnifierManager.cs

```diff
@@ -172,7 +178,27 @@ public void ApplyMagnification()
         {
             Shader.SetGlobalFloat(MagnificationPropertyName, 1.0f - Magnification);
         }
-
+        public void GetMousePos()
+        {
+            if (inLensMode)
```

Comment:
Just in case there is no main camera.

```suggestion
            if (inLensMode && Camera.main != null)
```